Deploy alpha version of our website to the master

ilyassakhanov opened the pull request:
Since we already have a working website and have something to show to the end-user, we could merge it to master.

ilyassakhanov committed:
Modify script to work with input validation script

ilyassakhanov committed:
Delete unused library and pointless commented code

ilyassakhanov committed:
Remove a line that was intended for testing

ya-iris committed:
Merge pull request #64 from Slemanof/045_empty_menus

045 empty menus

ilyassakhanov committed:
Add a row for a link of usual menu of a restaurant, replaced cost_for_2 with price_range

ilyassakhanov committed:
Modify an inserting script to insert a link to usuall menu of a restaurant, change cost_for_2 to price range

ilyassakhanov committed:
Modify a script that works with Zomato API to retireve a link to usual menu of a restaurant, cost_for_2 changes to price_range

ya-iris committed:
Merge pull request #65 from Slemanof/063_database_tweaks

063 database tweaks

Stanley008 committed:
Comming Soon - Wordpress

Wordpress comming soon landing webpage was created. WP files.

Stanley008 committed:
Preview of Comming Soon

Preview of Comming Soon website was created and pushed to the wordpress application.

Stanley008 committed:
Logos

Several logos were adjusted, with transparent background and needed resolution for WP.

riccardosl committed:
fixed old functions

riccardosl committed:
fixed old functions

ilyassakhanov committed:
Remove an unused list

ilyassakhanov committed:
Add a user table

ilyassakhanov committed:
Fix the typo in the user table, add missing semicolumn

ilyassakhanov committed:
Add a script that inserts user data and also encrypts/hashes sensible information about that user

ilyassakhanov committed:
Uncomment  useful part of the code

ya-iris committed:
Flask Application

Python Flask supported file with connection to MySQL database and MySQL queries

ya-iris committed:
Website Template

HTML template containing the page structure and MySQL fetched data incorporated in it

ya-iris committed:
Static Files

Static files including CSS, JavaScript and Images

ilyassakhanov committed:
Rename a insertUser to insert_user to make it follow snake case like everywhere in this project

riccardosl committed:
Merge pull request #70 from Slemanof/019_user_database

019 user database

ya-iris committed:
Wrong File has been committed

ya-iris committed:
HTML templates 

HTML files containing page structure and MySQL data incorporated in there

ilyassakhanov committed:
Update requirements.txt

ya-iris committed:
Wrong CSS file has been submitted

ya-iris committed:
CSS file

Styling of the page

ilyassakhanov committed:
Move a lines for connection to the database inside add method

ya-iris committed:
Removing duplicates

ya-iris committed:
Removing commented out code

ya-iris committed:
Enabling maximization of all restaurant cards

ya-iris committed:
Changing the name of template to render

ya-iris committed:
Cleaning the code

ya-iris committed:
Cleaning the code

ya-iris committed:
Removing duplicate code

Slemanof committed:
Merge pull request #72 from Slemanof/019_user_database

Update requirements.txt

ya-iris committed:
Bug fix

ya-iris committed:
Rename Flask Application/app.py to flask_application/app.py

This is done for the

ya-iris committed:
Update requeirements.txt

Adding requirements for implementation of flask_application in frontend-with-implemented-backend branch

ya-iris committed:
Duplicate file deletion

This duplicate file has been created accidentally

ya-iris committed:
Python file with optimized function

The route function has been refactored to reduce its Cognitive Complexity to maintain it. Multiple separate if else statements have been reduced to an optimal loop with single if and else statements

ya-iris committed:
Adapting queries for boolean values

The additional_options is changed for retrieving 'restaurant' table rows where 'vegetarian', 'vegan', 'gluten_free', 'credit_card' and 'takeaway' are boolean fields with value 1 as positive value. Previously, the queries were made for values of string type

ilyassakhanov committed:
Change data type of a timing to JSON insead of varchar(255)

ilyassakhanov committed:
Change the output data type from string to json dictionary

ilyassakhanov committed:
Change quotes from single to double around readtimings usage, remove conversion to string near readtimings usage

ya-iris committed:
Cleaning the code

ya-iris committed:
Merge branch 'frontend-with-implemented-backend' of https://github.com/Slemanof/easyeats.live into frontend-with-implemented-backend

ya-iris committed:
Merge pull request #77 from Slemanof/075_opening_hours_json

075 opening hours json

ilyassakhanov committed:
Rename the input_validation.py to menu_checker.py since it doesn't validate menus

ya-iris committed:
Convert menu_chcler output to json

With the help of JSON encoder the menu_checker output is in JSON format in both cases - when the menu is presenet and when it's not. In case the menu is present it is also first converted from string to dictionary using JSON decoder

ilyassakhanov committed:
Change data type of menu in database from VARCHAR to JSON

ilyassakhanov committed:
Add a special output into the file for bad menu input

ilyassakhanov committed:
Change the import file for checking menus, add lines to get_menu function for error handling

ya-iris committed:
Merge pull request #79 from Slemanof/076_menu_in_json

076 menu in json

ya-iris committed:
Fixed phone numbers extraction

The string with phone numbers from extracted data is converted into a list.  The save_phones function then saves the first one or two numbers if they exist in the phones_list. It's elements are inserted in the database

ya-iris committed:
Code optimization

The filtering procedures have been split into function which are created outside of the route function. Thus, filtering processes by cuisines, distance and dietary options are performed by 3 separate function respectively, which are called by each other in the route function. This has been done in order to not overload the route function and to structure the code

ya-iris committed:
Changing the naming

ya-iris committed:
Changing the naming

ya-iris committed:
Adding indentation in line 10

ya-iris committed:
Reformatting code

ya-iris committed:
Removing duplicate code

ya-iris committed:
Redundant file deletion

ya-iris committed:
Redundant file deletion

ya-iris committed:
069 changed boolean to tinyint

donnelmandimika committed:
Add files via upload

Shows the all the restaurants collected on Google Maps. You can then selected restaurants with a red circle around them.

donnelmandimika committed:
Delete maps_connverter.py

donnelmandimika committed:
Delete map_plotter.py

donnelmandimika committed:
Map plotter

Shows the all the restaurants collected on Google Maps. You can then selected restaurants with a red circle around them.

ilyassakhanov committed:
Fix the regex string to match hours with one digit

ilyassakhanov committed:
Fix the regex string to match hours with one digit

ilyassakhanov committed:
Add an init file for python to recognize it as a package

ilyassakhanov committed:
Add an init file for python to recognize it as a package

ilyassakhanov committed:
Add test for findtime function inside readtimings, one assertion for put_timings_order

ilyassakhanov committed:
Add test for findtime function inside readtimings, one assertion for put_timings_order

ya-iris committed:
Removing redundant files

ilyassakhanov committed:
Add one more asser for a fuction that puts opening hours for input separated by comma

ya-iris committed:
Removing log in details

ilyassakhanov committed:
Merge pull request #83 from Slemanof/081_change_boolean_to_tinyint

081 changed boolean to tinyint

ilyassakhanov committed:
Merge pull request #80 from Slemanof/069_partial_phone_numbers_mysql

069 partial phone numbers

ilyassakhanov committed:
Merge pull request #87 from Slemanof/frontend-with-implemented-backend

062 website template supported by backend

ilyassakhanov committed:
Change the test case with single digit for hour

ilyassakhanov committed:
Add a test for opening hours with days using interval

ilyassakhanov committed:
Add a test get opening hours with interval with czech week days

ilyassakhanov committed:
Change the error messafe got tests to contain names of the finctions instead of desciption of that function, add a test for put_timings_standalone

ilyassakhanov committed:
Add an error message for test_put_timings_standalone

ilyassakhanov committed:
Add an error message with better description to test_put_timings_interval, add a test for put_timings_standalone with czech days of the week

ilyassakhanov committed:
Add an error message for test_put_timings_standalone

ilyassakhanov committed:
Resolve merge conflict

ilyassakhanov committed:
Add a test for sanitize_zomato_timings

ilyassakhanov committed:
Add a test case for sanitize_zomato_timings with Midnignts

ilyassakhanov committed:
Add a test for word Noon and a word to in zomato timings

DeC0rtez committed:
Adding tests

DeC0rtez committed:
Deleting redundant lines

DeC0rtez committed:
adding error messages

ilyassakhanov committed:
Fix regex for matching days of the week

ilyassakhanov committed:
Add tests for get_opening_hours

riccardosl committed:
fixed test_cusines

ilyassakhanov committed:
Change a name for a folder with tests

ya-iris committed:
Opening the "Usual menu" in a new tab

Target attribute has been added to the <a> tag containing the link to the restaurant's usual menu.
The target attribute specifies where the linked document will open when the link is clicked. Its value has been set to "_blank" , thus, the linked document will open in a new tab

ilyassakhanov committed:
updates get_menu function to the latest version

ilyassakhanov committed:
Update the dependencies to other files

ya-iris committed:
Renaming the folder

Flask Application folder has been renamed to flask_application for further easer use

ya-iris committed:
Renaming the folder

Flask application folder has been renamed to flask_application

ya-iris committed:
Merge branch 'develop' into 092_usual_menu_presentation

ya-iris committed:
Removing Flask debug mode

Slemanof committed:
Merge pull request #90 from Slemanof/085_readtimings_test

085 readtimings test

riccardosl committed:
removed a comment line

fixed merge issues

riccardosl committed:
removing old zomato-api.py

Slemanof committed:
Merge pull request #100 from Slemanof/testing-hide-api2

It applies workflow automated tests

riccardosl committed:
Merge pull request #99 from Slemanof/098_remove_flask_debug_mode

098 remove Flask debug mode

riccardosl committed:
Merge pull request #94 from Slemanof/092_usual_menu_presentation

092 Usual menu presentation

Slemanof committed:
Merge pull request #89 from Slemanof/test_menu_checker2

Test menu checker2

riccardosl committed:
Adding 2 tests for API and MySQL db

Slemanof committed:
Merge pull request #101 from Slemanof/testing

This merge will:
Update README.md with initial version.
Adds two tests:
   - Test API response #55 
   - Test DB connection #54

ilyassakhanov committed:
Delete dublicate folder

ilyassakhanov committed:
Delete duplicate folder

ilyassakhanov committed:
Fix import error for test

ilyassakhanov committed:
Update .yml file for tests to be run on every push

ilyassakhanov committed:
Change a primary key of a user table to autoincreasing id in integer

ilyassakhanov committed:
Add a junction table between restaurant and user

ilyassakhanov committed:
change values in restaurant_user to NOT NUll

ya-iris committed:
Merge pull request #104 from Slemanof/103_update_database

103 update database

ya-iris committed:
Adding recommendations function

The recommendation function is declared outside of route and is called in the route function in case the user hasn't chosen any filters and the query string is empty. If so, it is checked if the user has liked any restaurants - if the user_id is in the restaurant_user junction table, containing user ids and ids of the liked restaurants. If no restaurants have been liked, so the user_id is absent in the junction table, then all restaurants are fetched to the page according to descending order of their ratings. Else, up to 3 most frequent cuisines among the liked restaurants are identified and the most frequent values of dietary options and price range. Then, restaurants matching these criterias are found and fetched to the page. Once the user has applied any filters, restaurants based on them are fetched instead.

ya-iris committed:
Adapting html for recommended restaurants data

The data from recommendations function has different length, as in case of recommended restaurants, coordinates are not required to be called from database. In this case it is important to specify in which row of data set the cuisine names are located, otherwise they won't be shown in the page.

ya-iris committed:
Merge pull request #96 from Slemanof/078_fix_menu_updater

078 fix menu updater

ilyassakhanov committed:
Add unique and not null not email and password in the database

ilyassakhanov committed:
Change email from unique to primary key along with id

ilyassakhanov committed:
Make restaurant_id and user_id in table restaurant_user as compount primary keys to avoid duplicates

ilyassakhanov committed:
Make id and email as a primary key in user table

ya-iris committed:
Merge pull request #108 from Slemanof/107_fix_email_password

107 fix email password

ya-iris committed:
Adding function for likes and unlikes

Function like_and_unlike is created outside of the route and is called in the route function in case the request method is POST when the like button is clicked. The function receives id of the restaurant from the like button of this restaurant's card. In case this id is together with the current user id in the restaurant_user junction table, it means the restaurant already has been liked and should be instead unliked, so it is deleted from the junction table. Otherwise, it is being liked and added to the table.

The check_liked_restaurants function is returning the list of liked restaurants of the current user to the template where it is used to identify the state of the like button.

The main route has been renamed to /home for easier identification

ya-iris committed:
Adding js for like button functionality

The favourites,js file is responsible for sending data to the application about like buttons being clicked and for reloading the page in order to update appearance of the buttons

ya-iris committed:
Adding like button functionality

The favourites.js file is responsible for sending data to the application about the like buttons being clicked and for reloading the page in order to update the buttons' appearance

ya-iris committed:
Changing the like button

There is created a condition for the like button based on the list of liked resturants of the current user, passed to the template from the /home route. If the id of the current restaurant is present in this list, the like button appearance is set to a filled star, while if it is not, the star is empty. The use of corresponding images is implemented with the help of svg's and sprite

In addition, at the end of the html file the favourites.js file is connected

ya-iris committed:
Adding styling to the like button

ya-iris committed:
Adding images for like button

ya-iris committed:
Styling the like button

ya-iris committed:
Adding function for liking restaurants

The like_and_unlike function is created outside of the the route and is called inside the route function in case the request method is POST so the button is clicked. Through the POST method the restaurant id is passed from the clicked like button to the function. It is then checked whether this restaurant has been liked - whether this restaurant's id is together with the current user's id in the restaurant_user table. If it is, then the click on the button leads to deleting the restaurant from favourites, otherwise, it is added to the junction table with favourites. 

The check_liked_restaurants function returns a list of liked restaurants' ids of the current user. This list is then passed to the template where it is used to define the like button appearance based on the presence of the restaurant in this list.

In addition, the route name has been changed to /home for easier identification

ya-iris committed:
Like button variations

The like button appearance depends on the conditions. In case the restaurant id is in the list of liked restaurants' ids, the button is given the appearance of a filled star, otherwise - an empty star. For this purpose, svg and sprite are used

At the bottom of the file, favourites.js is connected for the button functionality

ya-iris committed:
Removing duplicate of cursor initialization

ya-iris committed:
Changing current_user value

This change is useful for avoiding further merge conflicts with parallel branches

ya-iris committed:
Adapting template to new route name

The route name has been changed to /home, so the search button should be configured to perform its action for this route as well

ya-iris committed:
Making conditions for displaying data

There have been created conditions for showing the rows from the restaurant data set.
The html structure for distance and time data is only created under the condition this data is present in the dataset. In case the users have not specified their location, nothing appears then.

The same result is implemented for dietary options. In this case, HTML depends on values of these fileds and HTML structure is only initiated in case of positive value - 1.

The phone numbers fields as well are shown only in case there values are not 0

For credit_card and takeaway fields HTML structure is  always created but its contents depend on the negative or positive value of these fields.

In addition, in case the user has applied different filters for the combination which no restaurants hav been found, a text informing about this outcome is created in the place of restaurant cards

Stanley008 committed:
Typo Sort Fix

The typo in Sort by dropdown form was fixed correctly to "form".

ya-iris committed:
Fixing small amount of recommendations

After the recommended data based on user likes is produced, it is checked whether it is not too small. So, in case the number of recommended restaurants is smaller then 3, the restaurants will then instead be recommended just based on rating from high to low

Slemanof committed:
Merge pull request #115 from Slemanof/021_fav_and_like

021 fav and like

ya-iris committed:
Merge pull request #105 from Slemanof/017_recommendations

017 recommendations (backup version)

ya-iris committed:
Adding signup page template

The signup template includes 4 inputs for the user's name, email, password and its confirmation. In addition there is a link to the login page in case the user already has registered. When the password is being entered, it is shadowed. It also has a set minimum and maximum length: 8 -20 characters. The email is automatically validated by checking if it includes @ and characters before and after this character

ya-iris committed:
Adding login page template

The login form icludes 2 inpits - for email and password. All fields are set to mendatory and the form can not be submitted without filling all of them. There is also a line to the signup page in case the user has not registered yet

ya-iris committed:
Adding styling for forms

ya-iris committed:
Importing required modules

ya-iris committed:
Adding signup route

In the route function user input from the signup form is received. For name and email fields escape_string is applied in order to escape special characters and preserve the database from sql injections. The user input is validated first throw checking if the password and confirmed password match. If they don't a message about it is send to the user. Otherwise, validation is taken to the next step - checking if the email is already in the database. If it is, again, the user receives a message about it in the page. If there is no such email in the database, the user can be added to it. For this case user password is hashed and then all credentials are added to the database. The user then gets a message about successful regestration.

ya-iris committed:
Adding login route

In the route function users' credentials are received from the login form. escape_string is applied to email to avoid sql injections. First, it is checked if such email exists in the database. If it does not, the user receives an error message in the login page. Otherwise, the entered password is validated an compared with the one from the database - in the same record with the varified email. If passwords do not match - the user receives an error message. Otherwise, an access token is generated. It is linked to this user by setting the user's id from the database as identity. The token is set in the cookie which is set in the response object. Response is then send to the browser where the cookie with the token appears. In the jwt configuration in the beginning of the file the location for sending this cookie has been set to /home route which is supposed to be a protected roite in the application and only accessable with jwt token from cookies. Cookies here are by default httpOnly. They can also be configured to be secure, but this confirguration can only be set to True in production mode for HTTPS connection. Here, in development mode, this configuration is now set to False. The life time of JWT has been set to 2 hours, after this period the token will expire. Cookies will only be available untill the browser is closed.

ya-iris committed:
Setting logout route

The users will be redirected to this route after they click the "Logout" button in the home page. The route function clears the cookies , so the protected route will not be accessable

ya-iris committed:
Protecting the home page

@jwt_required is added before the route function. It checks for the jwt availability in the cookies. If there is no toke, it has expired or there is no cookie, the content of the page will not be accessable.
If the token has been varifies, the page is accessable.
Here the user identity is extracted from the token, which is the user's id. This id is then passed to the recommendations and like functions

ya-iris committed:
Adding jwt loader functions

If the access token is not valid for any reason (missing, expired, tampered with, etc) there is returned json in the format of {‘msg’: ‘why accessing endpoint failed’} along with an appropriate http status code (generally 401 or 422). With the jwt_manager loader functions it is possible to custmize this response.

expired_token_loader() is a function to call when an expired token accesses a protected endpoint

invalid_token_loader() is a function to call when an invalid token accesses a protected endpoint(for example, when cookie is missing)

After the token has expired/the user closed the browser and cookie has been deleted, if the user tries to reload the page or perform any actions leading to this(searching for restaurants, liking restaurants) it will be considered an unauthorised access attempt or access attempt with an expired token. Once it is detected, the user will be redirected to the login page to get access again

ya-iris committed:
Adding JavaScript for signup route

This file receives values from the signup form after its submission. The password is validated and for each unfollowed rule (inclusion of uppercase letters/lowercase letters/digits/special characters) the user will receive an error message about the missing components. If the password structure has been validated, the credentials are received in the /signup route via POST method. If they have been varified there as well, the route function sends a response, which is caught in JS. Same happens when an error message is sent from the route function. According to this message, an alert appears in the top of the page, having a corresponding design and contaning the message from backend.

If registration has passed successfully, the user is redirected to the login page.

In line 32 the url is used for localhost and should be changed in production mode

ya-iris committed:
Adding JavaScript for login route

JS for login route works simialar as for the signup route, yet does not perform validation. In case the response from the backend has been positive and the user has been varified, there is performed redirection to the home page. In case of errors with varifying the user, alerts with the corresponding messages are evoked.

Aagain,  the url in line 6 should be changed in production mode

ya-iris committed:
Adding logout button

ya-iris committed:
Adding JavaScript for logging out

After the logout button in the home page has been clicked, the user is redirected to the logout route. There, in the backend , cookies are cleared and a response message is send. If it is positive {'logout':True}, then the user is redirected to the login page

ya-iris committed:
Updating requirements

ya-iris committed:
Importing time module

ya-iris committed:
Merge branch 'develop' into 074_hide_unavailable_information_on_page

ya-iris committed:
Merge pull request #117 from Slemanof/074_hide_unavailable_information_on_page

074 hide unavailable information on page

ya-iris committed:
Adding template for redirection

This template will be rendered in the home page in case the token has expired or cookie is missin. It contains information for the users about their session being ended

ya-iris committed:
Rendering redirect template

If there is detected an attempt to reach the protected route wuthout cookie/token or with an expired token, the home page contents will not be available. Insted, another template (redirect.html) will be rendered, informing the user about the need to confirm their identity again

ya-iris committed:
Adding js for redirecting

This script is connected to the redirect template. After it is renedered in the home page, it will dissapear after the amount of time given in redirect.js and the user will be redirected to login page

ya-iris committed:
Closing span

ya-iris committed:
Adding svg arrow

ya-iris committed:
Input validation with regex

In signup and logout routes function input is checked for the valid structure as well as presence only of allowed characters

ya-iris committed:
Name length limit

ya-iris committed:
Removing password validation from JavaScript

riccardosl committed:
Adding Deployment Workflow actions

riccardosl committed:
update on identation

riccardosl committed:
update 2 to fix identation

ya-iris committed:
Adding instruction element to form

An element containing instructions about structure of password in signup form

ya-iris committed:
Adding JavaScript for instruction element

The instruction element will be 'hidden' or will emerge as the password input field is used or not

ya-iris committed:
Updating cuisine filters

Names and ids of cuisine filters have been changed based on cuisines available in the database

riccardosl reviewed (approved):
We should remove
flask_application/.idea/ folder

riccardosl committed:
Removing action that might create issue on server

Slemanof reviewed (approved):
I see that version ALPHA is ready to be deployed to the master.

ilyassakhanov committed:
Merge pull request #127 from Slemanof/126_adapt_frontend_for_cuisines_in_database

126 adapt frontend for cuisines in database

ya-iris committed:
Keeping user checks

User choices are now kepts in local storage, so after refreshing the page during the search, the checkboxes remain marked.
In case a refresh button is pressed, local storage is cleared, as well as filters and query string. In case the user logs out, local storage is cleared as well

ya-iris committed:
Smooth opening of cuisine filters sections

ya-iris committed:
Smooth opening of restaurant card

ya-iris committed:
Update style.css

Updating styling for restaurant card, search, refresh buttons and 'show more' button in restaurant card

ya-iris committed:
Update index.html

Creating refresh button, updating 'show more' button in restaurant card

ya-iris committed:
Merge branch 'develop' into 102_keep_users_checks

ya-iris committed:
Removing duplicates

ya-iris committed:
Merge pull request #130 from Slemanof/102_keep_users_checks

102 keep users checks

riccardosl committed:
Going throught the checklist

riccardosl committed:
Update on list

riccardosl committed:
Removing UUID requirments as discussed with team

ilyassakhanov committed:
Add a list of libraries used and one limitation

ilyassakhanov committed:
Add a list of libraries used and one limitation

ya-iris committed:
Changing daily menu structure

Daily menu data is extracted from the dictionary in restaurant data set. Dish names are extracted as keys, while prices - as values. Keys and values are put in separate divs then and a wrapped in one common div

ya-iris committed:
Styling daily menu structure

ya-iris committed:
Adding opening hours block

The div above the daily menu section has been split in 2, so the opening hours and the rest of information are in front of each other horizontally instead of being spread over the restaurant card vertically.
Alike the daily menu, opening hours are represented as key and value pairs from dictionary - days are keys and hours are values

ya-iris committed:
Stling opening hours and other information layout

The blocks with opening hours and the rest of information have been set in a row

ya-iris committed:
Duplicate removal

ilyassakhanov committed:
Merge branch 'develop' of github.com:Slemanof/easyeats.live into 128_complete_readme

ilyassakhanov committed:
Complete a list of dependencies, add a link for each repository used

ilyassakhanov committed:
Complete a list of dependencies, add a link for each repository used

ya-iris committed:
Merge pull request #131 from Slemanof/88_display_daily_menu_and_opening_hours

88 display daily menu and opening hours

ya-iris committed:
Merge pull request #123 from Slemanof/095_flask_login_infrastructure

095 flask login infrastructure

ilyassakhanov committed:
Merge branch '128_complete_readme' of github.com:Slemanof/easyeats.live into 128_complete_readme

ilyassakhanov committed:
Add a gif file for login example

ilyassakhanov committed:
Wrong branch

ilyassakhanov committed:
Add a usage example for Login

ya-iris committed:
Restructuring the application

ya-iris committed:
Merge branch 'develop' of https://github.com/Slemanof/easyeats.live into develop

ya-iris committed:
Removing authorization details

ilyassakhanov committed:
Add a login example to README

ilyassakhanov committed:
Fix display of Login example

ilyassakhanov committed:
Fix Login example

ilyassakhanov committed:
Add a title for Login example

ya-iris committed:
Finishing restructarization

The routes are located in different files in folder 'app' together with the file where the application is initialized - __init__.py. The application is run with the help of run.py

ya-iris committed:
Merge branch 'develop' of https://github.com/Slemanof/easyeats.live into develop

ilyassakhanov committed:
Add a Sign up example to examples in readme

jandvorak-dot committed:
Optimizing Connections

jandvorak-dot committed:
removed self-assign

ilyassakhanov committed:
Add a gif for general usage example

ilyassakhanov committed:
Add a general usage exaple, delete Usage section

ilyassakhanov committed:
Add limitation description

jandvorak-dot committed:
removed unnecessary line

ilyassakhanov committed:
Merge pull request #133 from Slemanof/daily-menu-update

Optimizing Connections

ilyassakhanov committed:
Add a list of requirements

ya-iris committed:
Updating the title

ya-iris committed:
Updating the title

ya-iris committed:
Updating the title

ilyassakhanov committed:
Add a try except block to prevent updating code from stopping

ilyassakhanov committed:
Add a script that schedules update_menu script once per day

ilyassakhanov committed:
Merge branch 'develop' of github.com:Slemanof/easyeats.live into develop

Slemanof committed:
Adds description to dependencies sec of the README.md and list references.

Slemanof committed:
Updates the logo in README to latest version.

ilyassakhanov committed:
Add a reference to Flask_JWT_Extended

ya-iris committed:
Merge pull request #134 from Slemanof/128_complete_readme

128 complete readme

ya-iris committed:
Moving module level import to the top of file

ya-iris committed:
Removing debug mode

ya-iris committed:
Setting the app to its initial structure

Due to issues with setting the app up on server, the previous structure has been set

ya-iris committed:
Fixing email validartion

ya-iris committed:
Removing odd file

ya-iris committed:
Removing bcrypt

Bcrypt turned out to be not compatible with the application in production, so instead werkzeug.security tools are used for hashing the password

ya-iris committed:
Removing bcrypt from dependencies

ya-iris committed:
Removing bcrypt

ilyassakhanov committed:
Add a script that reads and returns an encryption key

ilyassakhanov committed:
Add a script to use a script to import encryption key

ilyassakhanov committed:
Merge branch 'develop' of github.com:Slemanof/easyeats.live into develop

ilyassakhanov committed:
Remove config file for IDE

ya-iris committed:
Uncommenting code

ya-iris committed:
Update import_stuff.py
